packages/ra-data-simple-prisma/src/audit/types.ts

```diff
@@ -9,7 +9,7 @@
 // }
 
 export type AuditOptions = {
-  tableName?: string; //defaults to "audit_logs"
+  table?: any;
```

Comment:
you know it must have at least a create method, so this could be
```suggestion
  table?: { create: Function }
```